MEMBER_TRAVEL_FUND.md

```diff
@@ -94,6 +94,32 @@ As examples, the purpose of prior sponsored travels includes attending TC39 meet
 attending [OpenJS collaboration summits](https://github.com/openjs-foundation/summit),
 and mentoring [Code and Learn](https://github.com/nodejs/code-and-learn).
 
+## Event/Project specific requests
+
+While members from any OpenJS project can request approval for travel through
+the process outlined in the `Approval` section, there may be situations in which 
+a project wishes to manage the approvals an event or purpose instead of
```

Comment:
English is not my first language so I might be off, but I don't fully understand this phrase.

> to manage the approvals an event or

Reply:
Thanks for catching that. Will update.

Reply:
Fixed.